Uchu.Core/Scriptable/PauseableTimer.cs

```diff
@@ -0,0 +1,69 @@
+using System.Diagnostics;
+using System.Timers;
+
+namespace Uchu.Core.Scriptable
+{
+    /// <inheritdoc />
+    /// <summary>
+    ///     Class for a Pausable timer.
+    /// </summary>
+    public class PausableTimer : Timer
```

Comment:
timers from C# stdlib should be pausable(?)

Reply:
Only Stopwatch has that functionallity, this is a wrapper around it.